candidate-standard/clause_4_terms_and_definitions.adoc

```diff
@@ -48,6 +48,11 @@ SOURCE: https://www.iso.org/standard/32551.html[ISO 19133:2005]:
 
 region specified with a geographic position and radial distance
 
+[[spatio-temporal-data-definition]]
+=== spatio-temporal data
+
+data associated with a position in space and time
```

Comment:
Or 'data associated with position**s** in space and time'

Reply:
@dr-shorthair Thank you. We discussed that variant of the definition. Many use cases have multiple data at a single position, which underlies the EDR API. Having multiple pieces of data at multiple positions weakens the direct connection to coordinates, which is the essence of the API. We do not want to preclude data spread across multiple positions, but is the wrong emphasis.
What do you see as the advantage of the 'many to many' definintion?

Reply:
EDR API SWG 47 discussed, and agreed `position` preferred to `positions`. But position could be more than a point, such as a polygon, consistent with the ISO position definition.